molecule/commands.py

```diff
@@ -159,6 +159,8 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
+            kwargs['_env']['ANSIBLE_CALLBACK_PLUGINS'] = os.path.join(
+                sys.prefix, 'share/molecule/ansible/plugins/callback/idempotence')
```

Comment:
By setting it here, it looks like it negates the ability for us to override it in molecule.yml's [raw_env_vars](
https://github.com/metacloud/molecule/blob/master/docs/source/usage.rst#ansible
) section?